### PR TITLE
Fix OpenAPI schema for async responses

### PR DESCRIPTION
**Pull request by werwty**
closes #3668
https://pulp.plan.io/issues/3668

### CODE REVIEW

File: pulp_ansible/app/viewsets.py [hunk 1]
```diff
@@ -124,6 +126,9 @@ class AnsibleRemoteViewSet(RemoteViewSet):
     queryset = AnsibleRemote.objects.all()
     serializer_class = AnsibleRemoteSerializer
 
+    @swagger_auto_schema(operation_description="Trigger an asynchronous task to sync "
+                                               "ansible content.",
```
Comment (daviddavis) [inline pulp_ansible/app/viewsets.py]: s/ansible/Ansible/

File: pulp_ansible/app/viewsets.py [hunk 2]
```diff
@@ -156,6 +161,9 @@ class AnsiblePublisherViewSet(PublisherViewSet):
     queryset = AnsiblePublisher.objects.all()
     serializer_class = AnsiblePublisherSerializer
 
+    @swagger_auto_schema(operation_description="Trigger an asynchronous task to publish "
+                                               "ansible content.",
```
Comment (daviddavis) [inline pulp_ansible/app/viewsets.py]: here too

**Comment by dralley** [comment]
Test failures are due to smash changes, not anything this code is doing wrong.  I recommend disabling the test until further notice

**Comment by daviddavis** [comment]
Fixed the broken tests: https://github.com/pulp/pulp_ansible/pull/40

**Pull request merged**